src/material/schematics/ng-generate/theme-color/README.md

```diff
@@ -3,24 +3,64 @@
 ```shell
 ng generate @angular/material:theme-color
 ```
-
+## Background
 This schematic allows users to create new Material 3 theme palettes based
 on custom colors by using [Material Color Utilities](https://github.com/material-foundation/material-color-utilities).
+This is an alternative to using the available [predefined theme palettes](https://material.angular.io/guide/theming#prebuilt-color-palettes).
```

Comment:
This theming guide link no longer points to a specific section (in next.material.angular.io) - this and other spots in the README that need to be adjusted

The new theming guide doesnt really list out the prebuilts anymore. I feel like they might be outdated now that its so easy to add your own theme with mat.theme

Reply:
I was looking at the rc guide but just removed the rc in the url since the target for this PR is patch.

So this link for that is https://rc.material.angular.io/guide/theming#prebuilt-color-palettes, which does have the list of the prebuilt palettes. Should this still be removed?